test/DebugInfo/dropped-var.sil

```diff
@@ -1,4 +1,5 @@
  // RUN: %target-sil-opt --diagnose-unreachable -o - %s -sil-stats-lost-variables 2>&1 | %FileCheck %s
+ // REQUIRES: CPU=aarch64
```

Comment:
Wouldn't it be much better to just hardcode the triple in the command line?

Reply:
Done